test/model/src/core/kv_table.cpp

```diff
@@ -356,6 +376,34 @@ kv_table::verify_cursor()
     return std::move(kv_table_verify_cursor(_data));
 }
 
+/*
+ * kv_table::fill_missing_column_fix_recnos --
+ *     Fill in missing recnos for FLCS to ensure that key rages are contiguous.
```

Comment:
```suggestion
 *     Fill in missing recnos for FLCS to ensure that key ranges are contiguous.
```

Reply:
Nice catch! Fixed in [cef253a](https://github.com/wiredtiger/wiredtiger/pull/10520/commits/cef253acb4839b848ce9d087b852fe52bc791320).